code/model/Translatable.php

```diff
@@ -1897,7 +1897,7 @@ public function transformFormField(FormField $field) {
 	 * @param FormField $originalField The original editable field containing the translated value
 	 * @return CheckboxField The field with a modified label
 	 */
-	protected function transformCheckboxField(CheckboxField $nonEditableField, CheckboxField $originalField) {
+	protected function transformCheckboxField(CheckboxField_Readonly $nonEditableField, CheckboxField $originalField) {
```

Comment:
Is this parameter even used? It doesn't seem to be even referenced inside the method body.